src/shared/components/floating-button/floating-button.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChevronBigTopIcon, FabIcon } from "@assets/icons";
+
+import * as styles from "./floating-button.css";
+
+const FloatingButton = () => {
+  const handleScrollToTop = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
+  };
+
+  return (
+    <div className={styles.floatingButtonContainer}>
+      {/* 위 버튼: 최상단 이동 */}
+      <button
+        type="button"
+        className={styles.floatingButtonTop}
+        onClick={handleScrollToTop}
+        aria-label="최상단으로 이동"
+        style={{ pointerEvents: "auto" }}
+      >
+        <ChevronBigTopIcon className={styles.iconTop} />
```

Comment:
아이콘의 width와 height는 컴포넌트를 불러오는 곳에서 처리해주세요!
```suggestion
        <ChevronBigTopIcon width={24} height={24} />
```
이런 식으로요!